packages/shared/tailwind.config.ts

```diff
@@ -152,6 +152,7 @@ export default {
       14: '0.875rem',
       16: '1rem',
       18: '1.125rem',
+      20: '1.25rem',
```

Comment:
Tried to use `rounded-[20px]` but it didn't seem to catch it :triggered:, so I added i here

Reply:
rounded-[1.25rem] proably worked?
But it's fine this is a guideline thing so thanks for adding.

Reply:
Don't think it worked with rem, pretty sure I tested it 🙈

Reply:
fml... I didn't try with `rem`, only `px`. It works with `rem`, but I'll keep the addition.